cmd/sfncli/sfncli.go

```diff
@@ -65,28 +75,28 @@ func main() {
 	// register the activity with AWS (it might already exist, which is ok)
 	sfnapi := sfn.New(session.New(), aws.NewConfig().WithRegion(*region))
 	createOutput, err := sfnapi.CreateActivityWithContext(mainCtx, &sfn.CreateActivityInput{
-		Name: name,
+		Name: activityName,
 	})
 	if err != nil {
 		fmt.Printf("error creating activity: %s\n", err)
 		os.Exit(1)
 	}
-	log.InfoD("startup", logger.M{"activity": *createOutput.ActivityArn, "worker-name": workerName})
+	log.InfoD("startup", logger.M{"activity": *createOutput.ActivityArn, "worker-name": *workerName})
 
 	// run getactivitytask and get some work
-	// getactivitytask itself claims to initiate a polling loop, but wrap it in a polling loop of our own
-	// since it seems to return every minute or so with a nil error and empty output
+	// getactivitytask claims to initiate a polling loop, but it seems to return every few minutes with
+	// a nil error and empty output. So wrap it in a polling loop of our own
 	ticker := time.NewTicker(5 * time.Second)
-OuterLoop:
+GetActivityTaskPollingLoop: // breaking out of for-select is not pretty :-/ https://golang.org/ref/spec#Break_statements
 	for {
 		select {
 		case <-mainCtx.Done():
 			log.Info("getactivitytask-stop")
-			break OuterLoop // :-/ https://golang.org/ref/spec#Break_statements
+			break GetActivityTaskPollingLoop
```

Comment:
Nit.  Possible alternatives:
- Use `return`
- Create an `isDone` flag and use it as conditional `for !isDone {...`

Reply:
used `Err()` method on the context to get rid of label break here 4e5829c